CHANGELOG.md

```diff
@@ -23,3 +32,7 @@ Repository structure changes include:
 
 For more information on internal releases prior to NASA open-source approval,
 see legacy-CHANGELOG.md.
+
+[v1.1.0]:(https://github.com/nasa/harmony-swath-projector/releases/tag/1.0.1)
+[v1.0.1]:(https://github.com/nasa/harmony-swath-projector/releases/tag/1.0.1)
+[v1.0.0]:(https://github.com/nasa/harmony-swath-projector/releases/tag/1.0.0)
```

Comment:
@owenlittlejohns this is actually different from how I did it on some of the other changelogs, but I like this better. this is from common changelog and takes you to the release instead of the diff of the changes.